src/main/java/org/prebid/server/vast/VastModifier.java

```diff
@@ -15,14 +15,22 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class VastModifier {
 
-    private static final String IN_LINE_TAG = "<InLine>";
-    private static final String IN_LINE_CLOSE_TAG = "</InLine>";
-    private static final String WRAPPER_TAG = "<Wrapper>";
-    private static final String WRAPPER_CLOSE_TAG = "</Wrapper>";
-    private static final String IMPRESSION_CLOSE_TAG = "</Impression>";
+    private static final Pattern WRAPPER_OPEN_TAG_PATTERN =
+            Pattern.compile("<\\s*wrapper(>|\\s+.*?>)", Pattern.CASE_INSENSITIVE);
```

Comment:
`"<\\s*wrapper(?:>|\\s.*?>)"`
Same for others:
- capturing group -> non-capturing-group
- `\s+.*?` can be reduced to `\s.*?`